packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -256,8 +256,8 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
             _verifyExecBisectionRoot(_claim, _challengeIndex, parentPos, _isAttack);
         }
 
```

Comment:
No semver in this file?

Reply:
Hmm not in scope of this pr... Would defer to @clabby

Reply:
Probably best to bring discussion like this offline since it's not related to the pr